src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name, int radius, String color) {
+        super(name, color);
+        this.radius = radius;
+    }
+
+    @Override
+    public String drawFigure() {
+        return "Figure: " + this.getName()
+                + ", area: " + this.getArea()
+                + "color: " + this.getColor();
+    }
```

Comment:
do we need `this` keyword? I think I already wrote about this, why you did not remove 
 them? 🙂

Reply:
I dit.
I misunderstood you there, i then deleted some string which was there too.